### PR TITLE
kie-issues#527: Fix @kogito-apps/form-details: The embedded monaco-editor doesn't show the proper syntax highlighting

### LINKED ISSUE #527: Fix @kogito-apps/form-details: The embedded monaco-editor doesn't show the proper syntax highlighting
Once you generate custom forms with the **form-generation-tool** they can be edited in the *Runtime Tools Quarkus Extension* allows editing them on the *@kogito-apps/form-details* component.

The component uses the monaco editor to allow the edition, but it it's missing the syntax highlighting. 

This could be an issue on how the extension is bundled.

Steps to Reproduce:

1. Generate any type of forms using *form-generation-tool* for any BPM Quarkus project in kogito-examples (for example [process-usertasks-quarkus-with-console](https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/process-usertasks-quarkus-with-console))
2. In the project `pom.xml` add a dependency to the `runtime-tools-quarkus-extension`
3. Start the Kogito project in dev mode (`mvn clean quarkus:dev`)
4. Open the Dev ui (`http://locahost:8080/q/dev`) and click on the *Forms* link inside the *Runtime Tools Quarkus Extension*
5. Once in the forms list, click in any form and you'll be brought to the *form-details* view, on the left side you'll se a monaco-editor. It will show up without any kind of syntax highlighting.

### PR DESCRIPTION
**Pull request by jomarko**
Partially closes kiegroup/kie-issues#527

Because the mentioned ticket affects two modules:
- [runtime-tools-dev-ui-webapp](https://github.com/apache/incubator-kie-kogito-apps/tree/main/ui-packages/packages/runtime-tools-dev-ui-webapp)
- and [runtime-tools-quarkus-extension-deployment](https://github.com/apache/incubator-kie-kogito-apps/tree/main/runtime-tools-quarkus-extension-parent/runtime-tools-quarkus-extension-deployment)

This PR fix the `runtime-tools-dev-ui-webapp`, however `runtime-tools-quarkus-extension-deployment` still needs to be addressed.

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/apache/incubator-kie-kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to replicate CI configuration locally?
</summary>

Build Chain tool does "simple" maven build(s), the builds are just Maven commands, but because the repositories relates and depends on each other and any change in API or class method could affect several of those repositories there is a need to use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
 
[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
</details>

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- for <b>pull request checks</b>  
  Please add comment: <b>Jenkins retest this</b>

- for a <b>specific pull request check</b>  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] tests</b>

- for <b>quarkus branch checks</b>  
  Run checks against Quarkus current used branch  
  Please add comment: <b>Jenkins run quarkus-branch</b>

- for a <b>quarkus branch specific check</b>  
  Run checks against Quarkus current used branch  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] quarkus-branch</b>

- for <b>quarkus main checks</b>  
  Run checks against Quarkus main branch  
  Please add comment: <b>Jenkins run quarkus-main</b>

- for a <b>specific quarkus main check</b>  
  Run checks against Quarkus main branch  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] quarkus-main</b>

- for <b>quarkus lts checks</b>  
  Run checks against Quarkus lts branch  
  Please add comment: <b>Jenkins run quarkus-lts</b>

- for a <b>specific quarkus lts check</b>  
  Run checks against Quarkus lts branch  
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] quarkus-lts</b>
 
- for <b>native checks</b>  
  Run native checks  
  Please add comment: <b>Jenkins run native</b>

- for a <b>specific native check</b>  
  Run native checks 
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] native</b>

- for <b>native lts checks</b>  
  Run native checks against quarkus lts branch
  Please add comment: <b>Jenkins run native-lts</b>

- for a <b>specific native lts check</b>  
  Run native checks against quarkus lts branch
  Please add comment: <b>Jenkins (re)run [kogito-apps|kogito-examples] native-lts</b>
 
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

<details>
<summary>
Quarkus-3 PR check is failing ... what to do ?
</summary>
The Quarkus 3 check is applying patches from the `.ci/environments/quarkus-3/patches`.

The first patch, called `0001_before_sh.patch`, is generated from Openrewrite `.ci/environments/quarkus-3/quarkus3.yml` recipe. The patch is created to speed up the check. But it may be that some changes in the PR broke this patch.  
No panic, there is an easy way to regenerate it. You just need to comment on the PR:
```
jenkins rewrite quarkus-3
```
and it should, after some minutes (~20/30min) apply a commit on the PR with the patch regenerated.

Other patches were generated manually. If any of it fails, you will need to manually update it... and push your changes.
</details>

### CODE REVIEW

File: ui-packages/packages/runtime-tools-dev-ui-webapp/webpack.common.js [hunk 1]
```diff
@@ -64,9 +64,11 @@ module.exports = {
         onEnd: {
           mkdir: ['./dist/resources/webapp/'],
           copy: [
+            { source: './dist/editor.worker.js', destination: './dist/webapp/' },
+            { source: './dist/editor.worker.js.map', destination: './dist/webapp/' },
             { source: './dist/*.js', destination: './dist/resources/webapp/' },
             { source: './dist/*.map', destination: './dist/resources/webapp/' },
-            { source: './dist/fonts', destination: './dist/resources/webapp/' },
+            { source: './dist/fonts', destination: './dist/webapp/fonts' },
```
Comment (jomarko) [inline ui-packages/packages/runtime-tools-dev-ui-webapp/webpack.common.js]: Without this change, the `ui-packages/runtime-tools-dev-ui-webapp` showcase fails do load `woff` files - see the kie-issues#527.

After this change icons are loaded
Screenshot 2023-09-21 114444

**Review by yesamer — APPROVED** [review]
Well done!!

**Comment by pefernan** [comment]
@yesamer wait, because I think this is still on progress :P

**Comment by jomarko** [comment]
@yesamer yes, @pefernan is correct, this is still draft, work in progress, here is the current state ()

1.) in `runtime-tools-dev-ui-webapp` we fixed missing colors and icons for `html` and `tsx` form editor (so we have a progress, yay!)
2.) in `runtime-tools-quarkus-extension-deployment`  is the situation still the same

About the second point, it is for sure related to copying assets to some path, we do not know proper path, from my work, I do not think all assets can be copied to the same path

My plan is to:
1.) keep `runtime-tools-quarkus-extension-deployment`  in the same broken state as it is currently, but remove duplicated files, do simple exercise, `grep -name envelope.js`  it will return three copies on my machine, I believe only `dev-static/webapp/envelope.js`  is needed
2.) once we minify files copied from `runtime-tools-dev-ui-webapp` , lets try to find their proper placement for `runtime-tools-quarkus-extension-deployment`

**Comment by jomarko** [comment]
@yesamer @pefernan hi, let me share the status

1.) in `runtime-tools-dev-ui-webapp` we fixed now colors and fonts
2.) in `runtime-tools-quarkus-extension-deployment`  is the situation still the same, I just decreased quite amount of files we copy from `runtime-tools-dev-ui-webapp` into `runtime-tools-quarkus-extension-deployment`, however changes in `runtime-tools-quarkus-extension-deployment/pom.xml` are not final .

About the second point, I tried to summarize our issue on quarkus zulip topic https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/loading.20static.20resources

The code we use in https://github.com/apache/incubator-kie-kogito-apps/blob/main/runtime-tools-quarkus-extension-parent/runtime-tools-quarkus-extension-deployment/src/main/java/org/kie/kogito/runtime/tools/quarkus/extension/deployment/DevConsoleProcessor.java#L58 is deprecated, if you check in IDE and seems quite different from documented `BuildStep` here https://quarkus.io/guides/writing-extensions#extension-defined-endpoints 

What is your suggestion please?

**Comment by kie-ci3** [comment]
**PR job** `#8` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/pull-request-config.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1885 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/8/display/redirect

**Test results:**
- PASSED: 1654
- FAILED: 3

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/8/testReport/org.kie.kogito.it.trusty/SpringBootTrustyExplainabilityEnd2EndIT/doTest/">org.kie.kogito.it.trusty.SpringBootTrustyExplainabilityEnd2EndIT.doTest</a></summary>
Container startup failed
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/8/testReport/org.kie.kogito.index.service/KeycloakOracleIndexingServiceIT/testUnauthorizedUserAccess/">org.kie.kogito.index.service.KeycloakOracleIndexingServiceIT.testUnauthorizedUserAccess</a></summary>
java.lang.reflect.InvocationTargetException
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/8/testReport/org.kie.kogito.jitexecutor.bpmn.api/BPMNValidatorResourceTest/validateModel_MultipleValidBPMN2/">org.kie.kogito.jitexecutor.bpmn.api.BPMNValidatorResourceTest.validateModel_MultipleValidBPMN2</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>

**Comment by kie-ci3** [comment]
**PR job** `#11` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1885 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/11/display/redirect

**Test results:**
- PASSED: 2740
- FAILED: 8

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/11/testReport/org.kie.kogito.jobs.service.resource/KeycloakJobServiceTest/___/">org.kie.kogito.jobs.service.resource.KeycloakJobServiceTest.(?)</a></summary>
java.lang.reflect.InvocationTargetException
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/11/testReport/org.kie.kogito.jobs.service.messaging/InfinispanMessagingApiTest/cancelJob/">org.kie.kogito.jobs.service.messaging.InfinispanMessagingApiTest.cancelJob</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/11/testReport/org.kie.kogito.jobs.service.repository.infinispan/InfinispanJobRepositoryTest/testFindAll/">org.kie.kogito.jobs.service.repository.infinispan.InfinispanJobRepositoryTest.testFindAll</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/11/testReport/org.kie.kogito.jobs.service.resource/KeycloakInfinispanJobServiceTest/___/">org.kie.kogito.jobs.service.resource.KeycloakInfinispanJobServiceTest.(?)</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/11/testReport/org.kie.kogito.trusty.ui/KeycloakTrustyUIServiceIT/___/">org.kie.kogito.trusty.ui.KeycloakTrustyUIServiceIT.(?)</a></summary>
java.lang.reflect.InvocationTargetException
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/11/testReport/org.kie.kogito.trusty.ui/StaticContentTest/testIndexHtml/">org.kie.kogito.trusty.ui.StaticContentTest.testIndexHtml</a></summary>
java.lang.reflect.InvocationTargetException
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/11/testReport/org.kie.kogito.trusty.service.common/KeycloakTrustyServiceIT/___/">org.kie.kogito.trusty.service.common.KeycloakTrustyServiceIT.(?)</a></summary>
java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors<br/> [error]: Build step io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor#startKeycloakContainer threw an exception: java.lang.RuntimeException: org.testcontainers.containers.ContainerLaunchException: Container startup failed<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.startKeycloakContainer(KeycloakDevServicesProcessor.java:247)<br/> at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/> at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br/> at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br/> at java.base/java.lang.reflect.Method.invoke(Method.java:566)<br/> at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)<br/> at io.quarkus.builder.BuildContext.run(BuildContext.java:281)<br/> at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)<br/> at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)<br/> at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)<br/> at java.base/java.lang.Thread.run(Thread.java:829)<br/> at org.jboss.threads.JBossThread.run(JBossThread.java:501)<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Container startup failed<br/> at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:349)<br/> at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:322)<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.lambda$startContainer$4(KeycloakDevServicesProcessor.java:372)<br/> at java.base/java.util.Optional.orElseGet(Optional.java:369)<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.startContainer(KeycloakDevServicesProcessor.java:397)<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.startKeycloakContainer(KeycloakDevServicesProcessor.java:195)<br/> ... 11 more<br/>Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception<br/> at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)<br/> at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:334)<br/> ... 16 more<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container<br/> at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:542)<br/> at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:344)<br/> at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)<br/> ... 17 more<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Timed out waiting for log output matching '.*Keycloak.*started.*'<br/> at org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy.waitUntilReady(LogMessageWaitStrategy.java:47)<br/> at org.testcontainers.containers.wait.strategy.AbstractWaitStrategy.waitUntilReady(AbstractWaitStrategy.java:52)<br/> at org.testcontainers.containers.GenericContainer.waitUntilContainerStarted(GenericContainer.java:953)<br/> at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:485)<br/> ... 19 more<br/>
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/11/testReport/org.kie.kogito.trusty.service.common.api/DecisionsApiV1IT/testGetExecutionById/">org.kie.kogito.trusty.service.common.api.DecisionsApiV1IT.testGetExecutionById</a></summary>
java.lang.reflect.InvocationTargetException
</details>

**Comment by kie-ci3** [comment]
**PR job** `#1` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1885 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/test/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/1/display/redirect

**Test results:**
- PASSED: 2775
- FAILED: 4

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/test/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/1/testReport/org.kie.kogito.it.jobs/ProcessTimerIT/PR_check___Build_projects___testTimers/">PR check / Build projects / org.kie.kogito.it.jobs.ProcessTimerIT.testTimers</a></summary>
1 expectation failed.<br/>JSON path data.Jobs[0].status doesn't match.<br/>Expected: is "EXECUTED"<br/>  Actual: SCHEDULED<br/>
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/test/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/1/testReport/org.kie.kogito.jobs.service.messaging/InfinispanMessagingApiTest/cancelJob/">org.kie.kogito.jobs.service.messaging.InfinispanMessagingApiTest.cancelJob</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/test/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/1/testReport/org.kie.kogito.jobs.service.repository.infinispan/InfinispanJobRepositoryTest/testFindAll/">org.kie.kogito.jobs.service.repository.infinispan.InfinispanJobRepositoryTest.testFindAll</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/test/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/1/testReport/org.kie.kogito.jobs.service.resource/KeycloakInfinispanJobServiceTest/___/">org.kie.kogito.jobs.service.resource.KeycloakInfinispanJobServiceTest.(?)</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>

**Review by yesamer — APPROVED** [review]
LGTM

**Comment by kie-ci3** [comment]
**PR job** `#12` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1885 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/12/display/redirect

**Test results:**
- PASSED: 364
- FAILED: 8

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/12/testReport/org.kie.kogito.quarkus.outbox/OutboxIT/___/">org.kie.kogito.quarkus.outbox.OutboxIT.(?)</a></summary>
org.testcontainers.containers.ContainerLaunchException: Timed out waiting for container port to open (172.17.0.1 ports: [8083, 39767] should be listening)
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/12/testReport/org.acme.travels/ApprovalsProcessIT/PR_check___Build_projects___testApprovalProcess/">PR check / Build projects / org.acme.travels.ApprovalsProcessIT.testApprovalProcess</a></summary>
java.lang.reflect.InvocationTargetException
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/12/testReport/org.acme.travels/ApprovalsRestIT/testStartApprovalUnauthenticated/">org.acme.travels.ApprovalsRestIT.testStartApprovalUnauthenticated</a></summary>
Failed to load ApplicationContext
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/12/testReport/org.acme.travels/ApprovalsRestIT/PR_check___Build_projects___testStartApprovalAuthorized/">PR check / Build projects / org.acme.travels.ApprovalsRestIT.testStartApprovalAuthorized</a></summary>
Failed to load ApplicationContext
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/12/testReport/org.kie.kogito.springboot.outbox/OutboxIT/___/">org.kie.kogito.springboot.outbox.OutboxIT.(?)</a></summary>
org.testcontainers.containers.ContainerLaunchException: Timed out waiting for container port to open (172.17.0.1 ports: [8080, 39859] should be listening)
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/12/testReport/org.acme.travels/ApprovalsProcessTestIT/testApprovalProcess/">org.acme.travels.ApprovalsProcessTestIT.testApprovalProcess</a></summary>
Failed to load ApplicationContext
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/12/testReport/org.acme.travels/ApprovalsProcessTestIT/testApprovalProcessViaPhases/">org.acme.travels.ApprovalsProcessTestIT.testApprovalProcessViaPhases</a></summary>
Failed to load ApplicationContext
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/12/testReport/org.acme/AcmeExchangeResourceIT/unauthorizedTest/">org.acme.AcmeExchangeResourceIT.unauthorizedTest</a></summary>
java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors<br/> [error]: Build step io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor#startKeycloakContainer threw an exception: java.lang.RuntimeException: org.testcontainers.containers.ContainerLaunchException: Container startup failed<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.startKeycloakContainer(KeycloakDevServicesProcessor.java:247)<br/> at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/> at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br/> at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br/> at java.base/java.lang.reflect.Method.invoke(Method.java:566)<br/> at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)<br/> at io.quarkus.builder.BuildContext.run(BuildContext.java:281)<br/> at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)<br/> at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)<br/> at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)<br/> at java.base/java.lang.Thread.run(Thread.java:829)<br/> at org.jboss.threads.JBossThread.run(JBossThread.java:501)<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Container startup failed<br/> at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:349)<br/> at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:322)<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.lambda$startContainer$4(KeycloakDevServicesProcessor.java:372)<br/> at java.base/java.util.Optional.orElseGet(Optional.java:369)<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.startContainer(KeycloakDevServicesProcessor.java:397)<br/> at io.quarkus.oidc.deployment.devservices.keycloak.KeycloakDevServicesProcessor.startKeycloakContainer(KeycloakDevServicesProcessor.java:195)<br/> ... 11 more<br/>Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception<br/> at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)<br/> at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:334)<br/> ... 16 more<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container<br/> at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:542)<br/> at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:344)<br/> at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)<br/> ... 17 more<br/>Caused by: org.testcontainers.containers.ContainerLaunchException: Timed out waiting for log output matching '.*Keycloak.*started.*'<br/> at org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy.waitUntilReady(LogMessageWaitStrategy.java:47)<br/> at org.testcontainers.containers.wait.strategy.AbstractWaitStrategy.waitUntilReady(AbstractWaitStrategy.java:52)<br/> at org.testcontainers.containers.GenericContainer.waitUntilContainerStarted(GenericContainer.java:953)<br/> at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:485)<br/> ... 19 more<br/>
</details>

**Review by pefernan — COMMENTED** [review]
+1, thanks @jomarko

File: ui-packages/pnpm-lock.yaml [hunk 2]
```diff
@@ -5536,7 +5536,7 @@ packages:
     engines: {node: '>= 10.14.2'}
     dependencies:
       '@jest/types': 26.6.2
-      '@types/node': 20.4.1
+      '@types/node': 18.16.19
```
Comment (pefernan) [inline ui-packages/pnpm-lock.yaml:5539]: it's really curious how `pnpm` handles the versions. this kind ouf downgrades are surprising

**Comment by kie-ci3** [comment]
**PR job** `#13` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-apps -u https://github.com/apache/incubator-kie-kogito-apps/pull/1885 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/13/display/redirect

**Test results:**
- PASSED: 2775
- FAILED: 4

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/13/testReport/org.kie.kogito.index/ProcessDataIndexInfinispanIT/testProcessInstanceEvents/">org.kie.kogito.index.ProcessDataIndexInfinispanIT.testProcessInstanceEvents</a></summary>
1 expectation failed.<br/>JSON path errors doesn't match.<br/>Expected: null<br/>  Actual: <[{message=Exception while fetching data (/UserTaskInstanceCommentUpdate) : Index 0 out of bounds for length 0, locations=[{line=1, column=12}], path=[UserTaskInstanceCommentUpdate], extensions={classification=DataFetchingException}}]><br/>
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/13/testReport/org.kie.kogito.jobs.service.messaging/InfinispanMessagingApiTest/cancelJob/">org.kie.kogito.jobs.service.messaging.InfinispanMessagingApiTest.cancelJob</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/13/testReport/org.kie.kogito.jobs.service.repository.infinispan/InfinispanJobRepositoryTest/testFindAll/">org.kie.kogito.jobs.service.repository.infinispan.InfinispanJobRepositoryTest.testFindAll</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-apps-pr/job/PR-1885/13/testReport/org.kie.kogito.jobs.service.resource/KeycloakInfinispanJobServiceTest/___/">org.kie.kogito.jobs.service.resource.KeycloakInfinispanJobServiceTest.(?)</a></summary>
java.lang.RuntimeException: Failed to start quarkus
</details>

**Comment by yesamer** [comment]
@pefernan We can merge it after your approval, failing tests are unrelated.

**Comment by pefernan** [comment]
@yesamer already approved.. please go ahead!

**Review by pefernan — APPROVED** [review]

**Pull request merged**